pkg/util/workerqueue/workerqueue_test.go

```diff
@@ -81,3 +91,50 @@ func TestWorkerQueueHealthy(t *testing.T) {
 	assert.Equal(t, 0, wq.RunCount())
 	assert.EqualError(t, wq.Healthy(), "want 1 worker goroutine(s), got 0")
 }
+
+func TestWorkQueueHealthCheck(t *testing.T) {
+	t.Parallel()
+
+	health := healthcheck.NewHandler()
+	handler := func(string) error {
+		return nil
+	}
+	wq := NewWorkerQueue(handler, logrus.WithField("source", "test"), "test")
+	health.AddLivenessCheck("test", wq.Healthy)
+
+	server := httptest.NewServer(health)
```

Comment:
This is a great choice, as all the port handling between different tests isn't our responsibility.